src/mmw/js/src/core/views.js

```diff
@@ -79,11 +80,25 @@ var HeaderView = Marionette.ItemView.extend({
     },
 
     templateHelpers: function() {
-        var self = this;
+        var self = this,
+            currentActive = self.appState.get('active_page'),
+            isSelectArea = currentActive === coreUtils.selectAreaPageTitle,
+            isProjects = currentActive === coreUtils.projectsPageTitle,
+            isAnalyze = currentActive === coreUtils.analyzePageTitle,
+            modelPackageNames = _.pluck(settings.get('model_packages'), 'display_name'),
+            isModel = _.contains(modelPackageNames, currentActive),
+            isCompare = currentActive === coreUtils.comparePageTitle,
+            wasAnalyzeVisible = self.appState.get('was_analyze_visible'),
+            wasModelVisible = self.appState.get('was_model_visible'),
+            wasCompareVisible = self.appState.get('was_compare_visible');
```

Comment:
Oh, right. So we can nix that but then maybe just add another case along with the ones above?

* <img width="296" alt="screen shot 2017-03-01 at 6 05 48 pm" src="https://cloud.githubusercontent.com/assets/4165523/23485537/c1b8889e-fea9-11e6-9476-3d999f9925ad.png">